src/inverseRobot.js

```diff
@@ -25,6 +25,22 @@
  */
 function inverseRobot(robot) {
   // write code here
+  let reversed = {};
+  const arrayOfProp = [];
+
+  for (const prop in robot) {
+    arrayOfProp.push(robot[prop]);
```

Comment:
you can make your code prettier if you check here: got your object this property or not, and if it is got this property than you can immediately return null. This action  let you avoid the second loop